learning/tour-of-beam/frontend/lib/constants/assets.dart

```diff
@@ -20,5 +20,13 @@ String _getPath(String fileName, String format) {
   return '$format/$fileName.$format';
```

Comment:
I am not sure about this grouping. Assets are better grouped by component rather than by extension. But that's only if we want to group them in the first place.

Reply:
I've changed the code a bit, but this comment might still be actual. It just feels to store PNGs, SVGs and other files separately.

Reply:
That's only before you get an 80-screen app. :)